input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,499 @@
+            <th style="width: 25%;">International Patient Summary 1.1.0</th>
+            <th style="width: 25%;">Element</th>
+            <th style="width: 25%;">Additional requirements</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td style="width: 25%;"><a href="StructureDefinition-au-core-allergyintolerance.html">AU Core AllergyIntolerance</a></td>
+            <td style="width: 25%;"><a href="https://hl7.org/fhir/uv/ips/STU1.1/StructureDefinition-AllergyIntolerance-uv-ips.html">Allergy Intolerance (IPS)</a></td>
+            <td style="width: 25%;">AllergyIntolerance.patient.reference</td>
+            <td style="width: 25%;">IPS requires minimum of 1.</td>
+        </tr>
+        <tr>
+            <td rowspan="2" style="width: 25%;"><a href="StructureDefinition-au-core-condition.html">AU Core Condition</a></td>
+            <td rowspan="2" style="width: 25%;"><a href="https://hl7.org/fhir/uv/ips/STU1.1/StructureDefinition-Condition-uv-ips.html">Condition (IPS)</a></td>
+            <td style="width: 25%;">Condition.clinicalStatus</td>
+            <td style="width: 25%;">IPS requires minimum = 1.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;">Condition.subject.reference</td>
+            <td style="width: 25%;">IPS requires minimum of 1.</td>
+        </tr>
+        <tr>
+            <td rowspan="2" style="width: 25%;"><a href="StructureDefinition-au-core-diagnosticresult.html">AU Core Diagnostic Result Observation</a></td>
+            <td rowspan="2" style="width: 25%;"><a href="https://hl7.org/fhir/uv/ips/STU1.1/StructureDefinition-Observation-results-uv-ips.html">Observation Results (IPS)</a></td>
+            <td style="width: 25%;">Observation.status</td>
+            <td style="width: 25%;">IPS requires fixed value 'final'.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;">Observation.subject.reference</td>
+            <td style="width: 25%;">IPS requires minimum of 1.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;"><a href="StructureDefinition-au-core-immunization.html">AU Core Immunization</a></td>
+            <td style="width: 25%;"><a href="https://hl7.org/fhir/uv/ips/STU1.1/StructureDefinition-Immunization-uv-ips.html">Immunization (IPS)</a></td>
+            <td style="width: 25%;">Immunization.patient.reference</td>
+            <td style="width: 25%;">IPS requires minimum of 1.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;"><a href="StructureDefinition-au-core-medication.html">AU Core Medication</a></td>
+            <td style="width: 25%;"><a href="https://hl7.org/fhir/uv/ips/STU1.1/StructureDefinition-Medication-uv-ips.html">Medication (IPS)</a></td>
+            <td style="width: 25%;">Medication.ingredient.strength</td>
+            <td style="width: 25%;">IPS requires data type profile <a href="https://hl7.org/fhir/uv/ips/STU1.1/StructureDefinition-Quantity-uv-ips.html">Quantity (IPS)</a>.</td>
```

Comment:
Should say "IPS requires requires UCUM for coded quantity units.."

Reply:
Done